content/docs/operating/configuration.md

```diff
@@ -309,18 +309,21 @@ feature to replace the special `__address__` label.
 
 ### `<dns_sd_config>`
 
-A DNS-SD configuration allows specifying a set of DNS record names which
-are periodically queried to discover a list of targets (host-port pairs). The
-DNS servers to be contacted are read from `/etc/resolv.conf`.
+A DNS configuration allows specifying a set of DNS domain names which are
```

Comment:
I'd still keep some reference to service discovery here, even if you don't want to mention DNS-SD verbatim. Like "DNS-based service discovery configuration"?